### ISSUE TITLE
[Editor] Warning when compiling in Angular v9 project

### ISSUE THREAD

**Issue by PetarMetodiev** [telerik/kendo-angular#2740]
**Describe the bug**
When the `EditorComponent` is added to an **Angular CLI** project(either manually or using the schematics) with Angular 9 the following warning is thrown:

```
WARNING in Entry point '@progress/kendo-angular-editor' contains deep imports into 
'/project/node_modules/@progress/kendo-editor-common/prosemirror/view', 
'/project/node_modules/@progress/kendo-editor-common/prosemirror/model', 
'/project/node_modules/@progress/kendo-editor-common/prosemirror/state', 
'/project/node_modules/@progress/kendo-editor-common/prosemirror/commands', 
'/project/node_modules/@progress/kendo-editor-common/prosemirror/keymap', 
'/project/node_modules/@progress/kendo-editor-common/prosemirror/history', 
'/project/node_modules/@progress/kendo-editor-common/prosemirror/tables'. 
This is probably not a problem, but may cause the compilation of entry points to be out of order.
```

**To Reproduce**
1. Create a new **Angular CLI** project with Angular 9.
2. Add the `EditorComponent` using the [official instructions](https://www.telerik.com/kendo-angular-ui/components/editor/#toc-installation)

**Expected behavior**
No warnings are displayed.

**Comment by lumyslinski** [comment]
Hi.
I got the same problem. What is the exactly meaning of this problem? Is there any performance impact on the project?

**Comment by destus90** [comment]
Seems to me that we should resolve this ourselves, as described here https://github.com/angular/angular/issues/35615#issuecomment-592253747

**Comment by PetarMetodiev** [comment]
Thanks for the comment. We are working towards a fix for the issue. For now these warnings can be safely ignored.

**Comment by tsvetomir** [comment]
Fixed in v1.2.0
